osmaxx/api_client/API_client.py

```diff
@@ -112,20 +77,6 @@ def _make_request_authorized(self):
         # TODO: comply to jwt and check if key is valid, if it needs reinitialization etc
         self.headers['Authorization'] = 'JWT {token}'.format(token=self.token)
 
-    def _request_successful(self, response):
-        try:
-            response.raise_for_status()
-        except requests.HTTPError as e:
-            logging.error('API job creation failed.', str(e) + " response: " + str(response.__dict__))
-            return False
-        return True
-
-    def _extract_key_from_json_or_none(self, response, key):
-        if self._request_successful(response):
-            try:
-                json_response = response.json()
-                return json_response.get(key, None)
-            except ValueError:
-                logging.error('API job creation failed because of an JSON decoding Issue.', str(response.__dict__))
-                return None
-        return None
+
+def errors(http_error):
```

Comment:
I wonder whether a different name would make the usage more comprehensible. `reasons_for`, maybe?